Mathlib/RingTheory/Binomial.lean

```diff
@@ -253,10 +260,13 @@ instance Int.instBinomialRing : BinomialRing ℤ where
         ← Int.neg_ofNat_succ, ascPochhammer_smeval_neg_eq_descPochhammer]
       norm_cast
 
-noncomputable instance {R : Type*} [AddCommMonoid R] [Module ℚ≥0 R] [Pow R ℕ] : BinomialRing R where
+noncomputable local instance {R : Type*} [AddCommMonoid R] [Module ℚ≥0 R] [Pow R ℕ] :
```

Comment:
If this instance is a performance issue, I guess there should be a comment here too.